inc/saibuffer.h

```diff
@@ -484,21 +484,29 @@ typedef enum _sai_buffer_profile_attr_t
      * @brief Dynamic threshold for the shared usage
      *
      * The threshold is set to the 2^n of available buffer of the pool.
-     * Mandatory when SAI_BUFFER_POOL_THRESHOLD_MODE = SAI_BUFFER_THRESHOLD_MODE_DYNAMIC
```

Comment:
previously this was conditional depending on pool id passed SAI_BUFFER_PROFILE_ATTR_POOL_ID, so it was indirect dependency, did that changed ?